components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemOffer.vue

```diff
@@ -18,22 +25,40 @@ import { nftToOfferItem } from '@/components/common/shoppingCart/utils'
 import { usePreferencesStore } from '@/stores/preferences'
 import { useMakingOfferStore } from '@/stores/makeOffer'
 import MakeOffer from '@/components/offer/MakeOffer.vue'
+import GalleryItemPriceSection from '@/components/gallery/GalleryItemAction/GalleryItemActionSection.vue'
+import type { NFTOffer } from '@/components/useNft'
```

Comment:
should be under composable folder?

Reply:
I would leave it there for now, the same as other types NFTWithMetadata, NftResources

Reply:
oh yes, I mean the import should be

```diff
- import type { NFTWithMetadata, NftResources, NFTOffer } from '@/components/useNft'
+ import type { NFTWithMetadata, NftResources, NFTOffer } from '@/composables/useNft'
```

Reply:
updated, my bad : )